openbb_terminal/forex/oanda/oanda_model.py

```diff
@@ -1,6 +1,3 @@
-"""Oanda Model."""
```

Comment:
I did not do this. Is this a linter, or should I undo this?

Reply:
weird.  lets leave it since we do that everywhere